src/Yarp.ReverseProxy.Swagger/ReverseProxyDocumentFilter.cs

```diff
@@ -90,5 +112,72 @@ public void Apply(OpenApiDocument swaggerDoc, DocumentFilterContext context)
             swaggerDoc.SecurityRequirements = securityRequirements;
             swaggerDoc.Components = components;
         }
+
+        private static Dictionary<string, List<string>> GetPublishedPaths(IConfiguration configuration)
+        {
+            var validRoutes = new Dictionary<string, List<string>>();
+            var allConfigs = configuration.AsEnumerable().ToImmutableDictionary();
+            var allPaths = allConfigs.Where(q => q.Key.EndsWith("Match:Path"));
+            foreach (var (key, routeValue) in allPaths)
+            {
+                var methods = new List<string>();
+                for (var i = 0; i < 10; i++)
```

Comment:
Magic numbers 😃 What are they mean? I propose to add constants.

Reply:
changed.